README.md

```diff
@@ -217,6 +217,31 @@ For the third, restore the `cr` image by `-XX:CRaCRestoreFrom=PATH` option
 $JAVA_HOME/bin/java -XX:CRaCRestoreFrom=cr
 ```
 
+### CPU Features
+
+You may run `-XX:CRaCCheckpointTo=PATH` on a computer named NEW and
```

Comment:
This looks a bit like FAQ entry; for the general README file I would go with single sentence ('When running checkpoint and restore on different computers...' or something like that) and link, putting the contents here to the beginning of that doc. There, I'd start with a mild introduction saying that JVM can depend on the specific CPU.